modules/currentDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default function displayCurrentDate(dateContainer) {
+  setInterval(() => {
+    dateContainer.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  }, 1000);
+}
```

Comment:
- [ ] Why not make use of arrow syntax for functions as arrow functions are part of ES6 and it is a requirement in this project. Check out how to use the arrow function keyword [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
thanks for reviewing, for this function when i use arrow syntax i can't export it as default and at the same time i get linter error telling me that since i have one function in the file it should be default so escape the linter error i use the regular function syntax